### DISCUSSION TITLE
Specify preferred document order to render

### DISCUSSION THREAD

**Discussion by hectormz** [quarto-dev/quarto-cli#7298] [Q&A]
### Description

I mostly use `.ipynb` files to generate blog posts, but occasionally use `.qmd` files for simpler posts. At the moment I sync `.ipynb` files to `.qmd` (or `.py`) files with `jupytext` for version control (or using `.qmd` features with RStudio).

 If these files are in the same folder, I'd like to specify that `quarto` should prefer `.ipynb` files over `.qmd` files when available. Alternatively, I have been able to separate the file types in different sub folders and put an underscore in front of the `.qmd` filder so it gets ignored. But I don't like having to add an additional folder name blob to the blog post's url.

**Comment by mcanouil** [comment]
See #6518 especially:
- #6944

PS: could you search a bit before posting? Thank you.

**Reply by hectormz** [reply]
Thanks. I think this is different than #6944. I don't want to render all documents in a folder and choose the order they render. Rather, if a folder has a `.ipynb` only render that (and ignore any `.qmd`). If a folder only has a `.qmd`, render it.

**Reply by mcanouil** [reply]
Prefix the file with `_` then or define the render targets, see <https://quarto.org/docs/websites/index.html#render-targets>.
The behaviour you want is specific to you as many users use both type of files, what you are asking is never going to be implemented.

**Reply by hectormz** [reply]
Understood, thanks @mcanouil . I appreciate you fielding my issues and questions. I'll close this now
